<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDP</class>
 <widget class="QWidget" name="UDP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>131</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Receive</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QCheckBox" name="ckAssic">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>ASSIC显示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ckHex">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>HEX显示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="RevPort_Lable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>对方端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="listen_port"/>
    </item>
    <item>
     <widget class="QPushButton" name="Listen_Btn">
      <property name="text">
       <string>开始监听</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>250</y>
     <width>131</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Sender</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QCheckBox" name="ckSend_Assic">
      <property name="text">
       <string>ASSIC显示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ckSend_Hex">
      <property name="text">
       <string>HEX显示</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbSend_Ip">
      <property name="text">
       <string>对方IP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leSend_Ip"/>
    </item>
    <item>
     <widget class="QLabel" name="lbLocal_Port">
      <property name="text">
       <string>本地端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leLocal_Port"/>
    </item>
    <item>
     <widget class="QLabel" name="lbPosition_Port">
      <property name="text">
       <string>对方端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lePosition_Port"/>
    </item>
    <item>
     <widget class="QPushButton" name="Connect_Btn">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>508</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>UDP接受</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="tbRev"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leRev"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>UDP发送</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="tbSend"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leSend"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
